src/interactivity/win32/clipboard.hpp

```diff
@@ -41,8 +41,8 @@ namespace Microsoft::Console::Interactivity::Win32
 
         void StoreSelectionToClipboard(_In_ const bool fAlsoCopyFormatting);
 
-        void CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows, _In_ const bool copyFormatting);
-        void CopyToSystemClipboard(std::string stringToPlaceOnClip, LPCWSTR lpszFormat);
+        void CopyTextToSystemClipboard(const std::wstring_view text, const std::optional<std::string> htmlData, const std::optional<std::string> rtfData) const;
+        void CopyToSystemClipboard(const std::string_view stringToPlaceOnClip, LPCWSTR lpszFormat) const;

```

Comment:
@lhecker you're gonna conflict this, and because of that conflict I won't be able to easily backport to 1.19; thoughts? Want to maintain two copies of that PR?

Reply:
I'd be happy to split my PR into two to adapt for this one. 🙂